selfdrive/car/gm/radar_interface.py

```diff
@@ -16,9 +16,6 @@
 
 
 def create_radar_can_parser(car_fingerprint):
-  if car_fingerprint not in (CAR.VOLT, CAR.MALIBU, CAR.HOLDEN_ASTRA, CAR.ACADIA, CAR.CADILLAC_ATS, CAR.ESCALADE_ESV):
```

Comment:
The Buick Regal is not in this list, do we know if that was a mistake? If so, this is a good change. I wouldn't imagine it wouldn't have a radar, but we should check why it was left out.

Reply:
Think it was mistakenly forgotten to be added when we did the internal merge: https://github.com/commaai/openpilot/pull/528/files